pkg/oauthserver/authenticator/password/allowanypassword/anyauthpassword.go

```diff
@@ -33,5 +33,6 @@ func (a alwaysAcceptPasswordAuthenticator) AuthenticatePassword(username, passwo
 
 	identity := authapi.NewDefaultUserIdentityInfo(a.providerName, username)
 
-	return identitymapper.UserFor(a.identityMapper, identity)
+	user, ok, err := identitymapper.UserFor(a.identityMapper, identity)
```

Comment:
Please update `identitymapper.UserFor` to return `authenticator.Response`

Reply:
> Please update `identitymapper.UserFor` to return `authenticator.Response`

You can do that as a followup.  Think carefully about whether you think an identity should be able to choose an audience.  I think that's probably a mistake.